wiki/Getting-Started.md

```diff
@@ -66,6 +66,25 @@ When this happens, you usually see a black screen when trying to start niri from
 
 Also, on Intel graphics, you may need a workaround described [here](https://nixos.wiki/wiki/Intel_Graphics).
 
+To run Niri on NixOS is as simple as editing your configuration.nix to add
```

Comment:
For people using NixOS, looking for an option in https://search.nixos.org/options should be the first thing when installing some new software, as that usually takes care of a few extras in addition to installing the package.

I am not sure if it makes sense to add this here, unless it makes sense to also add instructions for other distros.